documentation/modules/ROOT/pages/architecture.adoc

```diff
@@ -30,13 +30,13 @@ As shown in the image, the {prodname} connectors for MySQL and PostgresSQL are d
 Kafka Connect operates as a separate service besides the Kafka broker.
 
 By default, changes from one database table are written to a Kafka topic whose name corresponds to the table name.
-If needed, you can adjust the destination topic name by configuring {prodname}'s {link-prefix}:{link-topic-routing}[topic routing transformation]. For example, you can: 
+If needed, you can adjust the destination topic name by configuring {prodname}'s {link-prefix}:{link-topic-routing}#topic-routing[topic routing transformation]. For example, you can:
```

Comment:
So why not change that to xref, when touching this anyways? My suggestion is this:

- No need to actively go and replace all link prefix ones, but
- new links should be xref
- any changed links should be updated to be xref

I.e. continuously improve the situation, so we can then clean up the remainder in a concerted action eventually.

Reply:
> change that to xref, when touching this

True. Point taken. I'm conscious of not wanting to break anything that's currently working, but I guess there's no reason to preserve these artifacts, and I appreciate that retaining them can cause confusion for other contributors.